standards/mobile_app_standards.md

```diff
@@ -0,0 +1,108 @@
+Mobile Application Standards
+============================
+
+Scope
+-----
+
+These standards apply to mobile application in-house development and management.
```

Comment:
I'd like to include something near the start about "don't use apps at all unless we can help it" - see https://gds.blog.gov.uk/2013/03/12/were-not-appy-not-appy-at-all/

Reply:
I've revised this to make it obvious that, when running down the options, you should stop as soon as you can.

Reply:
And basically I would agree with what the GDS are saying, and I have linked to the formal GDS advice in the "PWA" option.
PWAs should definitely be a preferred option, but we are already stuck with some non-PWA mobile apps for internal use, and that situation is likely to only change in one direction...